packages/web/src/scss/tools/_form-fields.scss

```
@@ -153,6 +154,16 @@
     &:focus-within {
         background-color: form-fields-settings.$input-background-color-state-active;
     }
+
+    @each $breakpoint-name, $breakpoint-value in tokens.$breakpoints {
+        @include breakpoint.up($breakpoint-value) {
+            --form-fields-border-radius: var(
```

i think we also add to these a prefix :) 
```suggestion
            --#{tokens.$css-variable-prefix}form-fields-border-radius: var(
```